src/shared/escaping.js

```diff
@@ -51,7 +51,9 @@ export function escape (info, options, escapeOptions) {
       }
     }
 
-    if (isString(value)) {
+    if (value === null) {
```

Comment:
This if is not necessary, the issue is that null also has a type of object. We can add an `isPureObject` method in `src/utils/is-type.js` which checks both type and if its not null and use that to check there instead of just isObject.

Reply:
> Thanks for this pr, I have some small remarks/requests.
> 
> Also please merge the master branch to exclude the dist folder again. If those files wont be removed after merging the master branch use `git reset dist/*` and if thats not enough `git checkout HEAD -- dist/*` to reset your branch back to the master state. Sorry for that

Just removed the changes in the `dist` folder :wink:

Reply:
I created a `isPureObject` function according to your suggestion.